pkg/daemons/control/server.go

```diff
@@ -174,22 +174,23 @@ func apiServer(ctx context.Context, cfg *config.Control, runtime *config.Control
 	argsMap["advertise-address"] = localhostIP.String()
 	argsMap["insecure-port"] = "0"
 	argsMap["secure-port"] = strconv.Itoa(cfg.ListenPort)
-	argsMap["bind-address"] = localhostIP.String()
-	argsMap["tls-cert-file"] = runtime.TLSCert
-	argsMap["tls-private-key-file"] = runtime.TLSKey
+	argsMap["bind-address"] = "0.0.0.0"
```

Comment:
why?

Reply:
This was a temporary change while working through how things need to connect, has been changed back to binding with localhost.